### DISCUSSION TITLE
devlife/2023-03-21-MySQL-accessdeniederror/

### DISCUSSION THREAD

**Discussion by giscus** [suakang17/suakang17.github.io#3] [Comments]
# devlife/2023-03-21-MySQL-accessdeniederror/

MySQL 8.0, MySQL workbench, Windows 10

https://suakang17.github.io/devlife/2023-03-21-MySQL-accessdeniederror/

**Comment by suakang17** [comment]
3번의 서비스는 제어판\시스템 및 보안\Windows Tools 에 위치해있다.
